LIB386/LIB_3D/P_OB_ISO.ASM

```diff
@@ -1284,7 +1284,7 @@ nosxmax:
 ; essai de correction tri par influence du y
 		sub	bp, bx
 
-		lea	ecx, [ebx+ebx]		; *2
+		lea	ecx, [ebx+ebx*1]	; *2
```

Comment:
Any particular reason for this change?

Reply:
it fixes the masm error:  "multiple base registers not allowed"

Reply:
That's weird. I wonder why this change would silence that error. To me it does not seem to make any sense.